src/main/java/blue/lapis/pore/command/CommandBase.java

```diff
@@ -0,0 +1,107 @@
+ * Pore
+ * Copyright (c) 2014-2015, Lapis <https://github.com/LapisBlue>
+ *
+ * The MIT License
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package blue.lapis.pore.command;
+
+import blue.lapis.pore.Pore;
+import blue.lapis.pore.impl.command.PoreCommandSender;
+
+import com.google.common.base.Optional;
+import org.spongepowered.api.Server;
+import org.spongepowered.api.text.Texts;
+import org.spongepowered.api.text.Text;
+import org.spongepowered.api.util.command.CommandCallable;
+import org.spongepowered.api.util.command.CommandException;
+import org.spongepowered.api.util.command.CommandResult;
+import org.spongepowered.api.util.command.CommandSource;
+
+import java.util.Collections;
+import java.util.List;
+
+public class CommandBase implements CommandCallable {
+
+    @SuppressWarnings("unused")
+    private Server server;
+    private Text desc;
+    private Text help;
+    private String permission;
+    private String usage;
+    private String command;
+    private List<String> aliases;
+
+    public CommandBase(Server server, String permission, String description, String help, String usage, String command, List<String> aliases) {
+        this.server = server;
+        this.permission = permission;
+        this.desc = Texts.of(description);
+        this.help = Texts.of(help);
+        this.usage = usage;
+        this.command = command;
+        this.aliases = aliases;
+    }
+
+    @Override
+    public Optional<CommandResult> process(CommandSource source, String arguments) throws CommandException {
+    	if (!arguments.isEmpty()) {
+    		command = command + ' ' + arguments;
+    	}
+    	
+    	Pore.getServer().dispatchCommand(PoreCommandSender.of(source), command);
+    	
+        return Optional.of(CommandResult.success());
+    }
+
+    @Override
+    public List<String> getSuggestions(CommandSource source, String arguments) throws CommandException {
+        return Collections.emptyList();

```

Comment:
No I'm not, but thanks. It is near to impossible to get all the arguments for their commands from all bukkit plugins you have installed.